RFC/RFC-11-hosted-preview-service.md

```diff
@@ -0,0 +1,131 @@
+
+Scope of the BETA: 
+- Open to users owning a Decentraland name 
+- Only one scene per Decentraland name would be allowed 
+- Scenes deployment to the preview server will have the same disk space limitations than the Content Servers hosted on the Catalyst network 
+- Besides scene size and owning a name, no other validations will be made as this is just a preview
+- Explorer mini map won't be displayed
+- Explorer skybox light can be set up for the scene
+- Display a Jump to Genesis Ctity button to leave the scene preview 
+- Validated by hosting a scene with the new onboarding experience
+
+
+#### The Solution 
+
+This implementation will allow content creators to deploy a scene in a preview server and join the scene with a maximum number of concurrent users. The solution is composed of three areas, the CLI, the backend infrastructure and the Explorer experience which will be detailed below. 
+
+#### Content Creators 
+
+From the Content Creators point of view, the experience will be very simple, only run the same deploy command but adding as a target the URL of the new preview server, e.g.  
+
+`dcl deploy -t https://example-url-for-dcl-preview-server`  
+ 
+ The output of this command should contain a link that can be shared and used to jump in to the scene preview, e.g. `https://play.decentraland.org/?PREVIEW_SERVER=...&SCENE=...&COMMS=....` 
+
+ #### Architecture 
+
+The Explorer client in order to work needs to resolve scenes, users identity (avatar, wearables, emotes, etc) and communications between peers. Unlike production where all of them are obtained from the same realm, here a Preview Content Server and a Comms Preview Server will be needed. 
+The scenes assets and information that the client needs will be resolved from the Preview Content Server and the communications orchestration between peers joining a preview will be handled by the Comms Preview Server. On the other hand, users identity will continue to be retrieved from a DAO Catalyst node.    
+For this to work, the client will receive several new parameters: **preview content server** from which to resolve the scene assets, **scene identifier**, and the **comms preview server URL**. 
+
+```mermaid
+flowchart TD
+    B["Explorer Client"]
+    C(Preview Content Server) -- scene-id --> B
+    D(Preview Comms Server)-->B
+    E("Catalyst/Realm") -- avatars -->B
+```
+
+#### BFF
+
+In order to run, the Client needs to connect to a BFF. In this case, the Preview Content Server will act as BFF implementing the endpoint `/about` documented in [RFC-10](https://github.com/decentraland/adr/blob/main/RFC/RFC-10-realm-description.md).
+
+
+This way the Client will load:
+- The scene from `configurations.scenesUrl` downloading the URNs from `configurations.scenesURNs`
+- The wearables from `content.publicUrl`
+- The profiles from `lambdas.publicUrl`
+- The Comms Server through BFF from `bff.publicUrl`
```

Comment:
I'm thinking that the `/about` endpoint may hold the truth about the minimap, taking it from the scene.json itself.

That way I'd add:
````suggestion
- The Communications Server through BFF from `bff.publicUrl`
- The minimap configuration (via scene.json) from `configurations.minimap`

##### `configurations.minimap` schema

```typescript
type MiniMapConfiguration = {
  enabled: boolean
  dataImage?: string   // defaults to https://api.decentraland.org/v1/minimap.png
  estateImage?: string // defaults to https://api.decentraland.org/v1/estateimap.png
}
```
````